src/main/java/task_tree_UI/TaskTreeUI.java

```diff
@@ -0,0 +1,58 @@
+package task_tree_UI;
+
+import UI.CentralWindow;
+import UI.TaskDependentWindow;
+import entity_factories.*;
+import entity_layer.*;
+import use_cases.display_task_tree.TaskTreeUIInterface;
+
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.*;
+import javax.swing.tree.DefaultTreeModel;
+import java.awt.*;
+import java.awt.event.*;
+
+
+
+/**
+ * A TaskTree UI class, that implements the JFrame interface by making a list of tasks displayed by a tree.
+ * Created: 11/15/2022
+ * Last updated: 11/26/2022
+ *
+ * @author Aayush Bhan
+ */
+
+
+public class TaskTreeUI extends CentralWindow {
```

Comment:
Good work slimming down here; while the file "looks" a little emptier, this definitely adheres more to the clean architecture, and keeps your code open for when you eventually implement your presenter and models.